.mci.yml

```diff
@@ -24,6 +24,8 @@ variables:
     ## cmake flag variables
     cmake_flags:
         ubuntu_cmake_flags: &ubuntu_cmake_flags -DCMAKE_CXX_FLAGS="-Wall -Wextra -Wno-attributes -Werror -Wno-error=missing-field-initializers"
+        ubuntu_boost_cmake_flags: &ubuntu_boost_cmake_flags -DBSONCXX_POLY_USE_BOOST=ON -DCMAKE_CXX_FLAGS="-Wall -Wextra -Wno-attributes -Werror -Wno-error=missing-field-initializers"
+        ubuntu_std_experimental_cmake_flags: &ubuntu_std_experimental_cmake_flags -DBSONCXX_POLY_USE_STD_EXPERIMENTAL=ON -DCMAKE_CXX_STANDARD=14 -DCMAKE_CXX_FLAGS="-Wall -Wextra -Wno-attributes -Werror -Wno-error=missing-field-initializers"
```

Comment:
Indeed! Sorry, I should have commented. It has std::experimental, but something is broken, which I was looking at:
https://evergreen.mongodb.com/task_log_raw/mongodb_cpp_driver_dev_ubuntu1404_debug_std_experimental_compile_and_test_3f3499ee80b26fe3c110340520b959dbf7df846a_16_01_14_21_52_21/0?type=T